### ISSUE TITLE
react-data-grid-addons includes React.js in it's compiled file

### ISSUE THREAD

**Issue by lord-xeon** [adazzle/react-data-grid#881]
> **HAVE YOU ALREADY SEARCHED FOR SIMILAR ISSUES? PLEASE HELP US OUT AND DOUBLE-CHECK FIRST!**

> **ALSO, PLEASE DON'T BE THAT PERSON WHO DELETES THIS TEMPLATE. IT'S HERE FOR A REASON.**

> **THANKS!**

**WHICH VERSION OF REACT ARE YOU USING?**
```
Officially Supported:
[ ] v0.14.x
```

```
Community Supported:
[x] v15.0.x
```

**WHICH BROWSER ARE YOU USING?**

```
Officially Supported:
[ ] IE 9 / IE 10 / IE 11
[ ] Edge
[x] Chrome
```

```
Should work:
[ ] Firefox
[ ] Safari
```

--------

**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**
When bundling an application  for deployment to production, if you've used anything under `react-data-grid-addons`, the following React warning is shown in the console:

```
main.js:48
Warning: It looks like you're using a minified copy of the development build of React. When deploying React apps to production, make sure to use the production build which skips development warnings and is faster. See https://fb.me/react-minification for more details.
```

**Expected/desired behavior**
No React warning included in the console when making a production build.

**Reproduction of the problem**
If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem.

**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**
Cleaning up the console, removing duplicate included libraries, streamlining the production

After spending all morning debugging my production build, and including/excluding each individual line, I narrowed it down to at least this import on `AutoCompleteEditor.js`

`const ReactAutocomplete       = require('ron-react-autocomplete');`

I couldn't make a ticket in that project, but the owner says he works for adazzle, so maybe it's something that you guys can work on together?

**Comment by supamanda** [comment]
This is a duplicate of https://github.com/adazzle/react-data-grid/issues/675

**Comment by malonecj** [comment]
Issue closed as duplicate #675 solved
